### PR TITLE
ci(#760): deploy allinone variant

### PR DESCRIPTION
**Pull request by aceforeverd**
* **Please check if the PR fulfills these requirements**

- The commit message follows our guidelines
- Tests for the changes have been added (for bug fixes / features)
- Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

File: .github/workflows/cicd.yaml [hunk 1]
```diff
@@ -271,6 +282,35 @@ jobs:
           MAVEN_TOKEN: ${{ secrets.OSSRH_TOKEN }}
           GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
 
+      - name: download shared libraries
+        if: github.event_name == 'push'
+        uses: actions/download-artifact@v2
+        with:
+          name: shared-library-${{ github.sha }}
+          path: java
```
Comment (tobegit3hub) [inline .github/workflows/cicd.yaml:286]: These so libraries will download to the sub-directory of `java`?

Comment (aceforeverd) [reply .github/workflows/cicd.yaml:286]: confirmed, all shared libraries appear in top directory inside jar
ref: https://s01.oss.sonatype.org/content/repositories/snapshots/com/4paradigm/openmldb/openmldb-native/0.4.0-allinone-SNAPSHOT/

**Review by tobegit3hub — APPROVED** [review]
LGTM

**Review by jingchen2222 — APPROVED** [review]
LGTM

**Comment by aceforeverd** [comment]
[github workflow reorganization](https://trello.com/c/upawl5mF/8-github-workflow-reorganization)

**Pull request merged**